notebooks/overlay-chop-multiple-participants.py

```diff
@@ -3,10 +3,99 @@
 
 # Suggest running this with: `caffeinate -i uv run overlay-chop-multiple-participants.py`
 
-# %%
+"""
+Video Processing for Eye Tracking Analysis
 
-import numpy as np
+This script processes eye tracking data from a research study that combines 
+video recordings with gaze tracking information for multiple participants. 
 
-# %%
+Key Functions:
+- Converting WMV video files to MP4 format
+- Processing eye tracking data from CSV files
+- Overlaying gaze positions on video recordings
+- Chopping videos based on annotation markers
+"""
 
-print("hello world")
+# %% Import Libraries and Configure Logging
+import pandas as pd
+import cv2
+from pathlib import Path
+from typing import Tuple
+from ivr_utils.ivr_utils import (
+    find_participant_files,
+    convert_wmv_to_mp4,
+    process_video,
+)
+
+import logging
+
+# Configure logging with timestamp, level and message
+logging.basicConfig(
+    level=logging.DEBUG,
+    format="%(asctime)s - %(levelname)s - %(message)s",
+    datefmt="%Y-%m-%d %H:%M:%S",
+)
+
+# Participants to process
+PARTICIPANTS = ["P48", "P42"]
+# Number of frames to process
+N_FRAMES_PROC = 30 * 120  # 30 fps * 120 seconds
+
+# Directory
+local_data_dir = Path.cwd().parent / "data"
+server_data_dir = Path("/Volumes/ritd-ag-project-rd01wq-tober63/SSID IVR Study 1/")
+output_dir = local_data_dir.joinpath("output/2025-02-10-test/")
+
+# Check if the server data directory exists
+assert server_data_dir.is_dir(), "Server data directory not found"
+
+# Create the output directory if it does not exist
+output_dir.mkdir(parents=True, exist_ok=True)
+
+# Eyetracking directory
+eyetracking_dir = server_data_dir / "Eyetracking"
+
+
+
+# %% Process each participant
+for PARTICIPANT_ID in PARTICIPANTS:
```

Comment:
To prepare for Myriad use we should probably wrap this up into a def main() and if __name__ == "main": statement.